packages/nextjs/index.ts

```diff
@@ -6,8 +6,11 @@ const CLIENT_PREFIX = "NEXT_PUBLIC_" as const;
 type ClientPrefix = typeof CLIENT_PREFIX;
 
 interface Options<
-  TServer extends Record<string, ZodType>,
-  TClient extends Record<`${ClientPrefix}${string}`, ZodType>
+  TServer extends Record<string, ZodType> = NonNullable<unknown>,
+  TClient extends Record<
+    `${ClientPrefix}${string}`,
+    ZodType
+  > = NonNullable<unknown>
```

Comment:
```suggestion
  TServer extends Record<string, ZodType>,
  TClient extends Record<`${ClientPrefix}${string}`, ZodType>
```

don't need defaults here